Controller/state.py

```diff
@@ -0,0 +1,10 @@
+#
+# file:   state.py
+# author: Nathan Burgers
+#
+
+from car import Car
+
+class State(object):
+    def __init__(self, cars = [Car(), Car()]):
```

Comment:
@CSE442/owners Hey everyone, this is an example comment for code review! You can say things like:

I don't think it makes sense for the state to be initialized with 2 `Car`s, default arguments should be the identity element for the particular argument.  A better default value for `cars` would be `[]`, which is the closest thing to an "identity" element for lists.  Please edit.